docs/user-guide.md

```diff
@@ -239,7 +240,62 @@ Pipelines can also trigger other pipelines upon completion. To do this, use the
         name: web-app-testing
 ```
 
-In the above example, the *ami-builder* pipeline runs every 7 days based on its schedule. When it completes, it executes the *my-web-app-pipeline* pipeline as defined in its *completion_trigger* property.
+Completion triggers can also be defined as part of the triggers expanded configuration. Take the above example for the ami-builder pipeline.
```

Comment:
Maybe we should propose the use of the `triggers` section going forward. And state that the `completion_trigger` is left in place for backward compatibility. What are your thoughts on this?

If we take this approach, we should flip the samples and show the full example using the `triggers` section and the minified version using the `completion_trigger`.